recipes/pyexplainer/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyexplainer" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c5ea63282519b87b5d3bafe41e6b6850e7f594631ecae5ae894c687afa0be470
+
+build:
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  skip: true  # [py<38]
```

Comment:
This package looks like it's pure python. If so, you should be able to change the `skip: true` to `noarch: python`. Then change the dependency on `python` under `host` and `run` to `python >=3.8`.

Reply:
okay let me do this